### DISCUSSION TITLE
0.12 RC1 on OpenMediaVault (Where is the TMP files stored at?)

### DISCUSSION THREAD

**Discussion by Iceman248** [blakeblackshear/frigate#5878] [Ask A Question] [answered]
I've been testing on OpenMediaVault, which uses Debian, using Portainer to install Frigate. It works fine, but I can't figure out where the /tmp/cache files are going. I've looked all over the partition, but nothing much at all in /tmp/cache, was able to locate files named as expected inside the container /tmp/cache directory, but I'm wanting to see where on the OMV system drive they are. Perhaps I should ask over there as well.

I see 2 video files per camera though, which is odd as my unRAID setup has the same size tmpfs volume, stable version though, and I have about 5 video files per camera.

**Comment by NickM-27** [comment]
It should be in RAM, but the portainer config would show for sure. If there are only 2 files vs 5 files in UnRaid that means the OMV machine is able to write the segments more quickly to storage than the UnRaid machine

**Reply by Iceman248** [reply]
What part of the portainer config should I be looking at? How does there only being 2 files vs 5 mean it must be writing faster? Thanks.

**Reply by NickM-27** [reply]
The portainer config will have (assuming it is matching the recommended docker config in the docs) a tmpfs mount on /tmp/cache. and if there are 5 files per camera compared to 2 then it is moving slower to move the files to hard storage because there are more.

**Reply by Iceman248** [reply]
I used the line about the tmpfs directly from the docs. It does seem to work, as it has seconds prior to object being detected as defined by me. I am just trying to find where these files are at outside of seeing them while consoled into the container.

**Reply by NickM-27** [reply, marked as answer]
Oh, you won't see them when outside the container because they are not mounted into any filesystem on the host. You can read more at https://docs.docker.com/storage/tmpfs/ and https://www.ecloudcontrol.com/where-are-docker-volumes-stored
